android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIImageView.java

```diff
@@ -140,10 +141,11 @@ public void loadImageFinished(int hash, Bitmap bitmap)
 						if (imgsrc == null) {
 							return;
 						}
-						if (imgsrc.hashCode() == hash
+						if (imgsrc.getUrl().hashCode() == hash
 							|| (imgsrc.getUrl() != null
```

Comment:
This will cause a crash if the `TiDrawableReference` object was not created from a URL. Such as via the static `fromResourceId()`, `fromBlob()`, and `fromFile()` functions.